### PR TITLE
Updating and expanding instructions for connecting to the radius network

### PR DESCRIPTION
**Pull request by viktorcsimma**
I've just helped someone connect to the network and for her, the only way her phone worked was by setting "Use system certificate" and the domain "wifi.eotvos.elte.hu". I've also made other changes and clarifications and included a link to the Windows guide I often use.

### CODE REVIEW

**Review by horcsinbalint — CHANGES_REQUESTED** [review]
Do not use external links

**Comment by viktorcsimma** [comment]
Why not? It worked for me a lot of times; we can still update it if it breaks sometime.

**Comment by horcsinbalint** [comment]
Someone might get scammed. No one likes to update because an external website seizes to operate

File: resources/lang/en/internet.php
Comment (horcsinbalint) [inline resources/lang/en/internet.php:1]: The domain name should be configurable in .env

**Comment by viktorcsimma** [comment]
Well, it would be great if someone could write an internal guide for the same; otherwise, I would insist on this. I think the benefits clearly outweigh the security risks.

@kdmnk what do you think?

**Comment by kdmnk** [comment]
If you say so, we can live with that. How often do you experience this specific issue? And yes maybe put that in .env

**Comment by viktorcsimma** [comment]
I think there have been ca. 5 cases where I followed (or simply sent) this tutorial and it helped. There was someone who had not had access for more than a year before I did that.

But putting it into .env is a good idea; I’ll take a look at it in the evening.

**Comment by horcsinbalint** [comment]
Does Windows care about it? I don't know, I usually don't work with such issues, but AFAIK if you simply forget the network and readd it, Windows does not care about what is the CA.
It is also a bit long description for the average user to follow.

**Comment by horcsinbalint** [comment]
Here is a Hungarian description from IIG: https://iig.elte.hu/dstore/document/3505/2020_wifi_installwin10.pdf

**Comment by horcsinbalint** [comment]
Should fix #402

**Comment by kdmnk** [comment]
Can we close this as duplicate to #452?

**Comment by horcsinbalint** [comment]
> Can we close this as duplicate to https://github.com/EotvosCollegium/mars/pull/452?

That was just a PR for the `internet-instructions` branch as it only contained suggestions for this PR.

**Review by horcsinbalint — COMMENTED** [review]
With the recent changes, I'm no longer blocking the PR. But please @kdmnk check on it before merging

**Review by horcsinbalint — APPROVED** [review]

**Pull request merged**
